lib/Main.js

```diff
@@ -50,10 +50,13 @@ function Main(config) {
     // So we can't create the StateLookup instance yet
     this._stateStorage = null;
 
+    var dbdir = config.dbdir || "";
     this._bridge = new Bridge({
         homeserverUrl: config.homeserver.url,
         domain: config.homeserver.server_name,
         registration: "slack-registration.yaml",
+        userStore: dbdir + "user-store.db",
```

Comment:
I prefer https://nodejs.org/dist/latest-v10.x/docs/api/path.html#path_path_join_paths simply because it stops people forgetting a /